pwa-shell/src/ui/menu/SearchBar.js

```diff
@@ -40,7 +52,13 @@ class SearchBar extends Component {
                       <FontAwesomeIcon icon="search" />
                     </InputGroupText>
                   </InputGroupAddon>
-                  <Input type="search" name="search" autoFocus />
+                  <Input
+                    type="search"
+                    name="search"
+                    value={this.state.searchTerm}
+                    onChange={e => this.onSearchValueChange(e.target.value)}
```

Comment:
Avoid using arrow function in `onChange`. Do `onchange={this.onSearchValueChange}`, then extract target value inside `onSearchValueChange`.

Reply:
You did told me about this. sorry I forgot